geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/parallel/ParallelGatewaySenderOperationsDUnitTest.java

```diff
@@ -1185,6 +1188,85 @@ public void testParallelGWSenderUpdateFiltersWhilePuttingOnOneDispatcThread() th
 
   }
 
+  /**
+   * Put entries in region after gateway sander is stopped. Count number of PQRM messages sent.
+   */
+  @Test
+  public void testDroppedEventsSignalizationToSecundaryQueueWhileSenderStopped() throws Exception {
+    int[] locatorPorts = createLNAndNYLocatorsReturnPrimitives();
+    int lnPort = locatorPorts[0];
+    int nyPort = locatorPorts[1];
```

Comment:
Since the `createLNAndNYLocatorsReturnPrimitives()` method is only used here, it's probably simpler to just inline it:
```
    int lnPort = vm0.invoke(() -> createFirstLocatorWithDSId(1));
    int nyPort = vm1.invoke(() -> createFirstRemoteLocator(2, lnPort));
```

Reply:
updated